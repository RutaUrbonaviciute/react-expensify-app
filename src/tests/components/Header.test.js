import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header.js';

describe('<Header />', () => {
  it('should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot()
  });
});


