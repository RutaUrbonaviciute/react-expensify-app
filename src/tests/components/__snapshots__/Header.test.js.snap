// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render header correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <Link
        className="header__title"
        to="/dashboard"
      >
        <h1>
          Expensify
        </h1>
      </Link>
      <button
        className="button button--link"
        onClick={[Function]}
      >
        Log Out
      </button>
    </div>
  </div>
</header>
`;
